docs/configuration/ccga.md

```diff
@@ -0,0 +1,15 @@
+# nfcore/RNAseq: CCGA Configuration
+
+The IKMB/CCGA Kiel currently maintains one HPC Cluster at Kiel University. 
+
+# CCGA
+
+To run the pipeline with the pre-configured environment, use `-profile ccga`. A full command line call will then look as follows:
+```
+nextflow run -profile ccga nf-core/RNAseq --reads '*_R{1,2}.fastq.gz' --genome GRCh37 
```

Comment:
Minor point: I would usually put `nextflow run nf-core/RNAseq -profile ccga` instead of `nextflow run -profile ccga nf-core/RNAseq`

I don't think it matters though, so down to personal preference I guess.